__tests__/search.test.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-env jest */
+const configPaths = require('../config/paths.json')
+const PORT = configPaths.port
+
+let browser
+let page
+let baseUrl = 'http://localhost:' + PORT
+
+beforeAll(async (done) => {
+  browser = global.browser
+  page = await browser.newPage()
+  done()
+})
+
+afterAll(async (done) => {
+  await page.close()
+  done()
+})
+
+describe('Search box', () => {
+  it('returns multiple results for a short query', async () => {
```

Comment:
We'd have to be able to build the site with a set of fixtures, which seems more effort than updating these tests when we add new things beginning with 'd'?

Reply:
Building the site with a set of fixtures would be one approach, but I was thinking more about asserting that certain results appear within the list (rather than asserting on the length of the list), and/or searching for a more specific substring than the letter 'd'.

Reply:
I suppose it depends what we want this test to give us confidence about, at the moment it's similar to a snapshot test.

If we assert the results contain certain results, we'd want to add tests that ensure we have not accidentally added 100s of results for example.

Reply:
Could we just assert that all of the search results have a word that start with the letter d?

Reply:
That sounds good